File: kattis/average_character.py
# https://open.kattis.com/problems/averagecharacter
# 1s 2048MB
# 1.9 Easy

# 55 chars
x=list(map(ord,input()))
print(chr(int(sum(x)/len(x))))


# 58 chars: Not short enough for Top 10 (32..48 chars)
# x=[ord(c)for c in input()]
# print(chr(int(sum(x)/len(x))))

File: kattis/average_character.rb
# https://open.kattis.com/problems/averagecharacter
# 1s 2048MB
# 1.9 Easy

# 45 chars Top 10 place 9  (in the Ruby Top 10 place 2, but there are only 3 entries)
x=$<.read[..-2]
$><<(x.sum/(x=~/$/)).to_i.chr

# 47 chars Top 10 place 10
# x=$<.read[0..-2]
# puts (x.sum/x.length).to_i.chr

